test/app/components/content_areas_component.html.erb

```diff
@@ -0,0 +1,5 @@
+<div>
+  <%= title %>
```

Comment:
It might be nice in this example to wrap each content area in a `div` with a class name like `title`, etc, to make this example feel more realistic.

Reply:
sure

Reply:
done